icu4c/source/test/intltest/unitstest.cpp

```diff
@@ -23,7 +26,8 @@ class UnitsTest : public IntlTest {
 
     void runIndexedTest(int32_t index, UBool exec, const char *&name, char *par = NULL);
 
-    void testConversions();
+    void testConversionCapability();
+    //  void testConversions(); // TODO(hugo): it doesnot pass.
```

Comment:
This now passes, probably due to the MeasureUnit fixes being live and merged now.
I tested with the commit here: https://github.com/younies/icu/pull/31 - accept that commit into this pull request to prove it's all working well, and to make the diffs a bit smaller?